### PR TITLE
Globalization

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to strongloop/loopback#2422

### CODE REVIEW

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@0candy @superkhau PTAL. Thanks!

And cc @rmg the mongodb server on platform amazon-5 is probably broken, test cases fail due to connection error. PTAL, thanks!
CI report: http://ci.strongloop.com/job/loopback-connector-mongodb/label=amazon-5/1453/console

**Comment by rmg** [comment]
@0candy good catch. I'll rebuild and redeploy the base image for that VM.

File: lib/mongodb.js [hunk 1]
```diff
@@ -94,7 +96,7 @@ function MongoDB(settings, dataSource) {
   this.debug = settings.debug || debug.enabled;
 
   if (this.debug) {
-    debug('Settings: %j', settings);
+    debug(g.f('Settings: %j', settings));
```
Comment (0candy) [inline lib/mongodb.js]: Don't have to globalize debug statements.

File: lib/mongodb.js [hunk 2]
```diff
@@ -137,12 +139,12 @@ MongoDB.prototype.connect = function(callback) {
     mongodb.MongoClient.connect(self.settings.url, self.settings, function(err, db) {
       if (!err) {
         if (self.debug) {
-          debug('MongoDB connection is established: ' + self.settings.url);
+          debug(g.f('{{MongoDB}} connection is established: %s' + self.settings.url));
```
Comment (0candy) [inline lib/mongodb.js]: Don't have to globalize debug statements.

File: lib/mongodb.js [hunk 3]
```diff
@@ -137,12 +139,12 @@ MongoDB.prototype.connect = function(callback) {
...
         }
         self.db = db;
       } else {
         if (self.debug || !callback) {
-          console.error('MongoDB connection is failed: ' + self.settings.url, err);
+          g.error('{{MongoDB}} connection is failed: %s %s' + self.settings.url, err);
```
Comment (0candy) [inline lib/mongodb.js]: Should use `,` not `+`

File: package.json [hunk 4]
```diff
@@ -23,10 +23,11 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "async": "^1.0.0",
+    "debug": "^2.1.1",
     "loopback-connector": "^2.2.0",
     "mongodb": "~2.1.21",
-    "async": "^1.0.0",
-    "debug": "^2.1.1"
+    "strong-globalize": "^2.5.7"
```
Comment (0candy) [inline package.json]: Latest is `2.5.8`

**Comment by jannyHou** [comment]
@0candy Thank you for reviewing. Feedback applied. PTAL again.

**Comment by 0candy** [comment]
LGTM

**Pull request merged**